pom.xml

```diff
@@ -26,11 +30,7 @@
             <groupId>org.slf4j</groupId>
             <artifactId>slf4j-simple</artifactId>
             <version>1.7.30</version>
-        </dependency>
-        <dependency>
-            <groupId>org.slf4j</groupId>
-            <artifactId>slf4j-log4j12</artifactId>
-            <version>1.7.30</version>
+            <scope>test</scope>
```

Comment:
We also need scoping outside of `tests` because this will be run separately from the `server`
```suggestion
```

Reply:
Thanks for the clarification regarding the architecture.

I removed the scope `test` and use logback as logging framework as it is a native implementation of slf4j.